feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenTest.kt

```diff
@@ -43,6 +43,36 @@ class SearchScreenTest {
         }
     }
 
+    @Test
+    @Category(ScreenshotTests::class)
+    @Config(fontScale = 0.5f)
+    fun smallFontScaleShot() {
```

Comment:
We are using check○○Shot() as naming🙏

Reply:
Sure! I'll handle it.

Reply:
https://github.com/DroidKaigi/conference-app-2023/pull/972/commits/be43cf32eb2093bdc55805dbc6c5d6c8c670683d
I have addressed it